### PR TITLE
Update Visual Design page to Wildcard page

### PR DESCRIPTION
**Pull request by jjinnii**
I will be separating out the content in the existing visual design page into two pages: Wildcard design system, and Visual design checklist. 
I think this change will help to make the content in the current page more consumable.

### CODE REVIEW

**Comment by jjinnii** [comment]
I noticed that all our design pages are written in an index.md file within a folder with that page name, but in the video for [creating a new page](https://handbook.sourcegraph.com/editing/adding-new-files) in our handbook, it didn't have the extra step of that index.md, which is why I created visual_design_checklist.md.

If I did that incorrectly, please let me know! It was my first time creating a new page so I may have messed up 😅

**Review by marisak — APPROVED** [review]
Hiya! The content of "Wildcard Design System" and "Visual design checklist" looks good to me.

Curious about a few things, listed below. None of these questions are blocking and are mroe about the organization of these pages than the content themselves.  
1. Is "visual design" a broader topic than the Wildcard design system? 
2. Building on (1), would it make sense to re-title the "Visual design" link in the Product design landing page (under Process) to "Wildcard design system" and have a separate link to other visual design resources? 
3. I didn't see a place where visual_design_checklist.md is linked. It seems like linking somewhere in /visual_design/index.md could be helpful, but that also depends on the answer to (1) above.

**Comment by jjinnii** [comment]
Hey @marisak! You're dead on with point 2, I had meant to do it but forgot :) "Wildcard design system" will be replacing the original "visual design" page. And. "Visual design checklist" is a new page altogether! Thanks for the callout

**Comment by jjinnii** [comment]
The failed check is due to having updated the link going to "Wildcard design system" as "wildcard_design_system.md", when it still reflects as "visual_design.md", but I don't see a way to update the link currently. I had thought the link automatically changes when I changed the heading of the page to "Wildcard design system." 

Would appreciate some help debugging. 😓

File: content/product/design/visual_design/index.md [hunk 1]
```diff
@@ -1,59 +1,44 @@
...
+
+- Modfiy the component in the Figmal file that requires the changed component by detaching it from the main instance.
+- Create a proposal frame with the following items:
+  - Proposal name and description
+  - Provide an example screenshot of the existing component
+  - Link to a Sourcegraph search showing affected areas of the product
+  - Screenshots of current product usage
+  - A design that represents the suggested change
+  - A clear verbal description of the suggested changes
+- @ mention a frontend engineer in the Figma file to request a change to an existing component. Engineering will approve, requests changes, or request that the designer uses an existing component
```
Comment (quinnkeast) [inline content/product/design/visual_design/index.md]:
```suggestion
- @mention a frontend engineer in the Figma file to request a change to an existing component. Engineering will approve, requests changes, or request that the designer uses an existing component
```

File: content/product/design/visual_design/index.md [hunk 2]
```diff
@@ -1,59 +1,44 @@
...
+- @ mention a designer and request a review
```
Comment (quinnkeast) [inline content/product/design/visual_design/index.md]:
```suggestion
- @mention a designer and request a review
```

File: content/product/design/visual_design_checklist.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Visual design checklist
+
+- Ensure spacing is consistent and matches the 8pt grid system
+- Ensure text, colors and other styles match and use existing styles in Wildcard, if possible.
+  - If adding new styles, ensure that they meet our [accessibility standards](https://handbook.sourcegraph.com/product/design/design-and-interaction-guidelines#accessibility-standards) and add them to Wildcard
+- Review designs for accessibility:
+  - Color blindness check
+  - Contrast check should meet AA standard for small text
+- Create dark mode compositions, if there are not existing dark mode designs in Wildcard
+- Consider preparing redlines or an interaction delivery writeup for the following :
```
Comment (quinnkeast) [inline content/product/design/visual_design_checklist.md]:
```suggestion
- Consider preparing redlines or an interaction delivery writeup for the following:
```

**Review by quinnkeast — APPROVED** [review]

**Review by rrhyne — APPROVED** [review]
LGTM

File: content/product/design/visual_design_checklist.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  - Margin and padding
+  - Animation
+  - Keyboard behavior
+- Due to the low number of mobile and table visitors (< 3%) design comps for these sizes are generally not required
```
Comment (AlicjaSuska) [inline content/product/design/visual_design_checklist.md]: I think this is an outdated statement. We do not target for mobile and tablet users that much but layouts need to be created in consideration for 1/2 and 1/3 browser windows (which effectively requires us to prepare responsive designs, at least for the main pages). In addition, we've recently decided to put more emphasis on responsiveness at Sourcegraph. My suggestion:

```suggestion
- Provide mockups to support the responsive behaviour. Even though the number of mobile and tablet visitors is relatively low, resizing the browser window to 1/2 and 1/3 width is a common use case. In addition, our objective is to provide a fully responsive platform. Note that there is no need to create a responsive version of every mockup. It's enough to include selected layouts that may be challenging to translate to smaller screen sizes.
```

**Review by AlicjaSuska — APPROVED** [review]
Thank you for making changes @jjinnii I made one suggestion regarding responsive layouts. Other than that, lgtm :)

**Pull request merged**